server/channels/store/sqlstore/post_store.go

```diff
@@ -2706,6 +2758,27 @@ func (s *SqlPostStore) GetRepliesForExport(rootId string) ([]*model.ReplyForExpo
 		return nil, errors.Wrap(err, "failed to find Posts")
 	}
 
+	postIds := make([]string, len(posts))
+	for i, post := range posts {
+		postIds[i] = post.Id
+	}
```

Comment:
By now I'd expect we would have generic utils to do this sort of simple manipulations.

Reply:
Removed this bit.